servlet-map/src/main/resources/META-INF/resources/spring-map-jsp/index.jsp

```diff
@@ -6,9 +6,9 @@
 <html>
 <head>
     <title>Oskari - ${viewName}</title>
-    <script type="text/javascript" src="//code.jquery.com/jquery-1.7.2.min.js">
+    <script type="text/javascript" src="//code.jquery.com/jquery-1.10.2.min.js">
```

Comment:
Hope this doesn't break anything.

Reply:
Tested with default Oskari setup and all seems to works. In front-end code Jstree library needs newer than 1.7.2 version of jQuery.